packages/portal-proto/src/features/set-operations/SetOperationChartsForCohorts.tsx

```diff
@@ -48,16 +48,22 @@ const SetOperationChartsForCohorts = ({
       filters: isCohortComparisonDemo
         ? buildCohortGqlOperator(cohortComparisonDemo1.filter)
         : buildCohortGqlOperator(getCohortFilterForAPI(cohorts[0])),
+      intent: "portal",
+      set_type: "ephemeral",
```

Comment:
<img width="1423" alt="Screenshot 2024-04-05 at 2 29 09 PM" src="https://github.com/NCI-GDC/gdc-frontend-framework/assets/4624053/7cf7e711-3a0d-4620-adca-6c2ff6ebfa8f">

the venn diagram analysis

Reply:
only wondering 'instant' would cause problems.

Reply:
I think it probably would. Each intersection of the venn diagram is its own request that uses the set ids that we create.

Reply:
ephemeral